x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alert_create_flyout.ts

```diff
@@ -294,20 +303,21 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await discardNewRuleCreation();
     });
 
-    it('should show error when es_query is invalid', async () => {
+    // Failing: See https://github.com/elastic/kibana/issues/126873
+    it.skip('should show error when es_query is invalid', async () => {
```

Comment:
this is the flaky test case, therefore moved `skip` from describe block to here.